geniza/corpus/management/commands/import_data.py

```diff
@@ -787,15 +812,7 @@ def get_source(self, edition, document):
         ed_orig = edition
 
         # set defaults for information that may not be present
-        title = volume = language = location = ""
-
-        # check for url and store if present
-        url_match = self.re_url.search(edition)
-        url = ""
-        if url_match:
-            # save the url, and remove from edition text, to simplify parsing
-            url = url_match.group("url")
-            edition = edition.replace(url, "").strip()
+        title = volume = language = location = journal = ""
```

Comment:
oh wow, I wouldn't have expected this to work, I guess python's smart enough for this.